docs/posts/ibis-bench/index.qmd

```diff
@@ -0,0 +1,1313 @@
+---
+title: "Ibis benchmarking: DuckDB, DataFusion, Polars"
+author: "Cody Peterson"
+date: "2024-06-23"
+image: "figure1.png"
+categories:
+    - benchmark
+    - duckdb
+    - datafusion
+    - polars
+---
+
+*The best benchmark is your own workload on your own data*.
+
+## Key considerations
+
+The purpose of this post is to explore some benchmarking data with Ibis. We'll
+compare a few modern query engines, explore the Ibis API as a great choice for
+each of them, and discuss the results.
+
+### The benchmark
+
+:::{.callout-important title="Not an official TPC-H benchmark"}
+This is not an [official TPC-H benchmark](https://www.tpc.org/tpch). We ran a
+derivate of the TPC-H benchmark.
+:::
+
+[The TPC-H benchmark](https://www.tpc.org/tpch) is a benchmark for databases
+and, [increasingly](https://docs.coiled.io/blog/tpch),
+[dataframes](https://pola.rs/posts/benchmarks)!  It consists of 22 queries
+across 8 tables. The queries are designed to test the performance of a database
+(or dataframe!) on a variety of tasks including filtering, aggregation, and
```

Comment:
```suggestion
across 8 tables. The queries are designed to test the performance of a query engine
on a variety of tasks including filtering, aggregation, and
```

Reply:
I disagree with this change -- TPC-H is clearly initially designed for databases, and I want to make the point (perhaps more explicitly somewhere) that Ibis brings the best of database technology including query engines to dataframes

Reply:
Fair enough, but the parenthetical seems out of place to me here then. Why point out the parenthetical if it's specifically designed for databases? Seems distractiing.

Reply:
This also isn't blocking.

Reply:
how's:

> The SQL (or dataframe) code representing the queries is
designed to test the performance of a query engine on a variety of tasks
including filtering, aggregation, and joins.

?

Reply:
I just don't like parentheticals. What value is it adding here?